src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+  (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
- */
+ **/
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
```

Comment:
Consider removing the comment '// write your code here' as it is no longer needed.

Reply:
I don`t know how it left I make a changes before commit                                                                                                        
      ![Screenshot_57](https://github.com/mate-academy/js_style-to-object/assets/123593139/08d2af5f-880a-4165-be26-345a6813312d)